### PR TITLE
APEXMALHAR-2034 Adding new Avro Module to encapsulate Container File to GenericRecord to POJO transformation

### PR DESCRIPTION
**Pull request by SaumyaMohan**

### CODE REVIEW

**Comment by tweise** [comment]
Please update existing PR instead of opening new PRs. See:
http://apex.apache.org/contributing.html#opening-pull-requests

Also it will be good to build this on top of #662 as it will eliminate dealing with old classes.

I would also like to see more info in the JIRA on how this work improves the Avro support (vs. just moving around code).

**Comment by sandeshh** [comment]
@tweise It is a simple fix, want to merge this quickly and move on to different tasks. We don't want to add any dependencies on different PRs.

**Comment by SaumyaMohan** [comment]
@tweise I've updated the JIRA to reflect the changes done as part of it. 
JIRA: https://issues.apache.org/jira/browse/APEXMALHAR-2034

File: avro/pom.xml [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+    </dependencies>
+
+</project>
```
Comment (sandeshh) [inline avro/pom.xml:260]: Add a new line

**Pull request closed without merging**